### ISSUE TITLE
Unable to connect to OFTC network over SSL

### ISSUE THREAD

**Issue by rtnpro** [martynsmith/node-irc#247]
Hi,

We are using node-irc to connect users at https://waartaa.com to IRC servers. We are generating self signed openssl certificates as mentioned at https://github.com/waartaa/waartaa/tree/master/app/private/certs and consuming the certificates at https://github.com/waartaa/waartaa/blob/master/app/server/lib/01_requirements.js#L32. This works perfectly well when connecting to servers like Freenode and Mozilla over SSL, but not for OFTC (irc.oftc.net). There's an ongoing discussion on this at https://github.com/waartaa/waartaa/issues/130#issuecomment-48127033.

I tried using the 'debug' mode to connect to OFTC server using SSL, and I was getting the following errors:
```
I20140707-20:22:02.931(0)? 7 Jul 20:22:02 - Disconnected: reconnecting
I20140707-20:22:02.931(0)? 7 Jul 20:22:02 - Waiting 2000ms before retrying
I20140707-20:22:02.932(0)? 7 Jul 20:22:02 - Connection got "end" event
```

Please help us resolve this issue. Once we figure out a way to connect to OFTC IRC server using node-irc over SSL, I will add a documentation at http://www.oftc.net/NickServ/CertFP/#unlisted-clients as well.

Thanks,
rtnpro

**Comment by Trinitas** [comment]
i get "SELF_SIGNED_CERT_IN_CHAIN"

**Comment by apeiron** [comment]
This is fixed in #304, and we will be testing this today and tomorrow with a release tomorrow evening assuming everything goes well.

**Comment by rtnpro** [comment]
Thanks a lot :+1:
